src/Sarif/CodeGenHints.json

```diff
@@ -4,7 +4,6 @@
       "kind": "DictionaryHint",
       "arguments": {
         "valueTypeName": "SerializedPropertyInfo",
-        "namespaceName": "Microsoft.CodeAnalysis.Sarif.Readers",
```

Comment:
>Microsoft.CodeAnalysis.Sarif.Readers [](http://example.com/codeflow?start=26&length=36)

This is to cleanup unused namespace in autogenerated codes.
"*.Properties" applies to all entities in Sarif, but only a few classes need the namespace "Microsoft.CodeAnalysis.Sarif.Readers".
Instead adding the namespace for all classes, just explicitly adding namespace for the classes require it.